Firmware.md

```diff
@@ -22,46 +22,28 @@ section for more information about the different firmware versions.
 
 ## Getting Started
 
- * TODO Put instructions here.
-
+If you are user, look at the 'Flashing prebuilt firmware' section.
 If you are a developer, look at 'Setting up a Developer Environment' section
-below. If you are user, look at the 'Flashing prebuilt firmware' section.
+below. 
 
 ### Flashing prebuilt firmware
-
```

Comment:
Why are you deleting all this information?

Reply:
It isn't useful.